packages/input-phone/src/CountrySelectorTrigger.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import emojiSupport from "detect-emoji-support";
+
+import ChevronDown from "./ChevronDown";
+
+const Trigger = styled.div`
+  padding: 0 4px 0 8px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  cursor: ${({ disabled }) => (disabled ? "default" : "pointer")};
+`;
+
+const CountrySelectorTrigger = ({ callingCode, flag, code, ...props }) => {
+  return (
+    <Trigger {...props}>
+      {emojiSupport() ? flag : `${code} `}+{callingCode}
```

Comment:
`emojiSupport` can be called on mount only

Reply:
`detect-emoji-support` already caches the results:
https://github.com/danalloway/detect-emoji-support/blob/0ceca07e9fd51077f372fd15d99eb0e4a142083b/lib/index.js#L3-L6